### PR TITLE
Create UI for New Game

### PR DESCRIPTION
**Pull request by tonyc856**
Draft/Skeleton of Create New Game UI

**New**
* CreateGame.js: main component for a player to join or create a new game
* Header.js: component representing the header on top of a page (the "Welcome" text in CreateGame.js in this case)

**UI Snapshot (Chrome):**
Desktop
Screen Shot 2020-05-20 at 10 24 29 AM

Mobile
Screen Shot 2020-05-20 at 10 25 27 AM

Ticket: #13

### CODE REVIEW

**Comment by reny1cao** [comment]
Looks good to me. Do we need to stick with the mockup? Like the color

**Comment by tonyc856** [comment]
> Looks good to me. Do we need to stick with the mockup? Like the color

Bonnie said to follow the mock up but can add our own creative design on top of it (like improving UX). As for colors, I'm using the default colors of Material UI's Button.

**Comment by ManpreetBhuller** [comment]
It looks nice!

File: client/src/pages/create_game/CreateGame.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import Grid from "@material-ui/core/Grid";
+import Input from "@material-ui/core/Input";
+import Typography from "@material-ui/core/Typography";
+
+import Header from "../common/Header";
+import "./CreateGame.css";
+
+function CreateGame() {
+  return (
+    <>
```
Comment (bonnieli) [inline client/src/pages/create_game/CreateGame.js]: curious - any reason for an extra element wrapper here?

Comment (tonyc856) [reply client/src/pages/create_game/CreateGame.js]: My apologies. I had 2 separate elements that needed to be wrapped before but not anymore. Thanks for pointing it out!

**Review by reny1cao — APPROVED** [review]

**Pull request merged**